requirements_dev.txt

```diff
@@ -41,7 +41,7 @@ ipython==5.3.0
 isort==4.2.5              # via flake8-isort
 jedi==0.10.2
 json-delta==2.0
-lxml==3.6.0
+lxml==3.9.1
```

Comment:
@cnlucas lxml version in `requirements-dev` are different `requirements.txt`. Are they meant to be like that?

Reply:
Whoops, thanks @pkfec